src/Controls/src/Core/Device.cs

```diff
@@ -54,9 +54,7 @@ public static TargetIdiom Idiom
 		/// <include file="../../docs/Microsoft.Maui.Controls/Device.xml" path="//Member[@MemberName='RuntimePlatform']/Docs" />
 		public static string RuntimePlatform => DeviceInfo.Platform.ToString();
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Device.xml" path="//Member[@MemberName='SetFlowDirection']/Docs" />
-		[EditorBrowsable(EditorBrowsableState.Never)]
-		public static void SetFlowDirection(FlowDirection value) => FlowDirection = value;
+		// [Obsolete]
 		/// <include file="../../docs/Microsoft.Maui.Controls/Device.xml" path="//Member[@MemberName='FlowDirection']/Docs" />
 		public static FlowDirection FlowDirection { get; internal set; }
```

Comment:
Because FlowDirection is per-window, we probably can make this not be a settable prop, but maybe `Windows[0].FlowDirection ?? LTR` so it is obvious that this is a proxy property.